extensions/oidc/runtime/src/main/java/io/quarkus/oidc/runtime/DefaultTenantConfigResolver.java

```diff
@@ -185,34 +186,22 @@ private String resolveStaticTenantId(RoutingContext context) {
             return tenantId;
         }
 
-        if (tenantResolver.isResolvable()) {
-            tenantId = tenantResolver.get().resolve(context);
-
-            if (tenantId != null) {
-                return tenantId;
-            }
-        }
-
-        if (pathMatchingTenantResolver != null) {
-            tenantId = pathMatchingTenantResolver.resolve(context);
-
+        for (var staticTenantResolver : staticTenantResolvers) {
+            tenantId = staticTenantResolver.resolve(context);
             if (tenantId != null) {
+                if (OidcUtils.DEFAULT_TENANT_ID.equals(tenantId)) {
```

Comment:
Returning `Default` must work as well, I think I have one of the tests doing it

Reply:
It would mean that tenant is in the `io.quarkus.oidc.runtime.TenantConfigBean#getStaticTenantsConfig`, which it is not.

Reply:
I think, I'll check, I have test for this.

Reply:
yeah, it only works because tenantConfigBean.getStaticTenantsConfig().get(tenantId) returns null so the default is assigned, clever, I didn't realize that. I'll drop this condition.

Reply:
Dropped, thanks.